### ISSUE TITLE
Dry run

### ISSUE THREAD

**Issue by igorkamyshev** [igorkamyshev/trona#201]
Add option for dry run.

**Comment by igorkamyshev** [comment]
Nope, I meant, this command should only show logs — what scripts are going to be executed. E.g., `name publish --dry-run` doesn't publish anything, but it shows a full info about publication.

**Comment by uka17** [comment]
@igorkamyshev 
So, command `yarn trona --dry-run` should show logs of potential execution.
Let's assume we have scripts:
- 1.sql
- 2.sql
- 3.sql

Case 1:
First execution of `yarn trona --dry-run` will just output all scripts considering execution order.
Case 2:
All subsequent executions `yarn trona --dry-run` after case 1 will not output anything (no any evolution is being applied)
Case 3:
2.sql was changed. `yarn trona --dry-run` will output degrade and evolution scripts of all upcomming files starting from 2.sql.

Correct?
